CONTRIBUTORS.md

```diff
@@ -32,6 +32,48 @@ This project and everyone participating in it is governed by this [Code of Condu
 - For consensus breaking changes, branch off of the `next` branch.
 - For hotfixes, branch off of `master`.
 
+### GitHub Tags
+We use tags on PRs and issues to categorize them. You should try to select a tag from each category below if possible. 
+
+**Timeline- and consensus- based tags:**
+- Next non-breaking change => `ship now`
+- Next breaking change => `ship next`
```

Comment:
"next: consensus"

Reply:
Yeah, I prefer something like `next stacks` or `next consensus` to `ship next`

Reply:
"ship consensus" to avoid using "next"?